docs/howto/custom-template-tags.txt

```diff
@@ -794,8 +794,10 @@ The ``__init__`` method for the ``Context`` class takes a parameter called
 This is not a very common situation, but it's useful if you're rendering a
 template yourself. For example::
 
+    from django.template import Context
+
     def render(self, context):
-        t = context.template.engine.get_template('small_fragment.html')
+        t = context.template.engine.get_template('small_fragment.html').template
```

Comment:
`Template` has not `template` attribute.

Reply:
@felixxm Sorry about that, I was testing it on `Django=3.2.7`. Thanks for your review!